blogposts/2021/release-3.32.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Sourcegraph 3.32 release'
+publishDate: 2021-09-23T10:00-07:00
+description: 'Sourcegraph 3.32 introduces Code Insights in Public Beta, a new search sidebar for navigating revisions, improved search results caching, and custom LSIF data retention policies.'
+tags: [blog, release]
+slug: 'release/3.32'
+published: false
+heroImage:
+socialImage:
+changelogItems:
+  - description: The search sidebar now shows a revisions section when all search results are from a single repository. This makes it easier to search in and switch between different revisions.
+    url: https://github.com/sourcegraph/sourcegraph/pull/23835
+    category: Search
+  - description: The various alerts overview panels in Grafana can now be clicked to go directly to the relevant panels and dashboards.
+    url: https://github.com/sourcegraph/sourcegraph/pull/24920
+    category: Admin
+  - description: '`allowGroupsPermissionsSync` in the GitHub authorization provider is now required to enable the experimental GitHub teams and organization permissions caching.'
```

Comment:
@iskyOS - do you think we need to provide any more context on what "experimental GitHub teams and organization permissions caching" is?

Reply:
Yes, that's a good idea — I'm going to add a link to the docs and give a little context here.

Reply:
```suggestion
  - description: '`allowGroupsPermissionsSync` in the GitHub authorization provider is now required to enable the experimental GitHub teams and organizations permissions caching. Permissions caching can significantly reduce the amount of time it takes to perform a full permissions sync.'
```